cmd/kubeadm/app/preflight/checks.go

```diff
@@ -357,3 +364,17 @@ func runChecks(checks []PreFlightCheck, ww io.Writer) error {
 	}
 	return nil
 }
+
+func TryStartKubelet() {
+	// If we notice that the kubelet service is inactive, try to start it
+	initSystem, err := initsystem.GetInitSystem()
+	if err != nil {
+		fmt.Println("[preflight] No supported init system detected, won't check if kubelet is running")
+	} else if !initSystem.ServiceIsActive("kubelet") {
+
+		fmt.Printf("[preflight] Starting the kubelet systemd service by running %q\n", "systemctl start kubelet")
+		if err := initSystem.ServiceStart("kubelet"); err != nil {
+			fmt.Println("[preflight] Couldn't start the kubelet service via systemd. Please start the kubelet service manually and try again.")
```

Comment:
Should we include the "err" here? Might be something relevant to the user in there.